src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -124,7 +124,7 @@ internal static object GetPrimitiveDefaultValue(Type type)
             switch (type.GetTypeCode())
             {
                 case TypeCode.Boolean:
-                    result = ScriptingRuntimeHelpers.False;
+                    result = False;
```

Comment:
Nit: I wonder if maybe we want to keep the type name prefix here.  The difference between "False" and "false" is very subtle.